src/app/hiv-care-lib/surge-report/surge-report-patientlist.component.ts

```diff
@@ -0,0 +1,166 @@
+import { Component, OnInit, Input } from '@angular/core';
```

Comment:
Change name of this file to reflect the suggested changes and all other relevant references.
`surge-report-patient-list.component.ts`